spec/ruby/core/enumerator/lazy/grep_v_spec.rb

```diff
@@ -0,0 +1,84 @@
+require File.expand_path('../../../../spec_helper', __FILE__)
+require File.expand_path('../fixtures/classes', __FILE__)
+
+describe "Enumerator::Lazy#grep_v" do
+  before(:each) do
+    @yieldsmixed = EnumeratorLazySpecs::YieldsMixed.new.to_enum.lazy
+    @eventsmixed = EnumeratorLazySpecs::EventsMixed.new.to_enum.lazy
+    ScratchPad.record []
+  end
+
+  after(:each) do
+    ScratchPad.clear
+  end
+
+  it "requires an argument" do
+    enumerator_class::Lazy.instance_method(:grep_v).arity.should == 1
+  end
+
+  it "returns a new instance of Enumerator::Lazy" do
+    ret = @yieldsmixed.grep_v(Object) {}
+    ret.should be_an_instance_of(enumerator_class::Lazy)
+    ret.should_not equal(@yieldsmixed)
+
+    ret = @yieldsmixed.grep_v(Object)
+    ret.should be_an_instance_of(enumerator_class::Lazy)
+    ret.should_not equal(@yieldsmixed)
+  end
+
+  it "sets nil to size" do
```

Comment:
I think this should be `it "sets #size to nil"` if I'm reading it right. @ahmadsherif I'm adding you Rubinius collaborators. I'll merge this. You can push an update to change the spec description.